include/nlohmann/detail/meta/type_traits.hpp

```diff
@@ -13,6 +13,7 @@
 #include <tuple> // tuple
 #include <type_traits> // false_type, is_constructible, is_integral, is_same, true_type
 #include <utility> // declval
+#include <cstddef> // byte
```

Comment:
Since you only need the header for `std::byte`, please only include it in case of

```cpp
#if defined(__cpp_lib_byte) && __cpp_lib_byte >= 201603L
```